### ISSUE TITLE
Building error on Debian with GR3.9

### ISSUE THREAD

**Issue by geosphere** [argilo/gr-dsd#20]
Kali-linux (equivalent Debian-tested), x86_64
Python 3.9.9
GR 3.9.4

```
[ 86%] Building CXX object lib/CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o
In file included from /opt/gr-dsd/lib/dsd_block_ff_impl.h:15,
                 from /opt/gr-dsd/lib/dsd_block_ff_impl.cc:12:
/opt/gr-dsd/dsd/include/dsd.h:25: warning: "__USE_XOPEN" redefined
   25 | #define __USE_XOPEN
      | 
In file included from /usr/include/x86_64-linux-gnu/c++/11/bits/os_defines.h:39,
                 from /usr/include/x86_64-linux-gnu/c++/11/bits/c++config.h:586,
                 from /usr/include/c++/11/cstddef:49,
                 from /usr/include/gnuradio/types.h:15,
                 from /usr/include/gnuradio/runtime_types.h:15,
                 from /usr/include/gnuradio/io_signature.h:15,
                 from /usr/include/gnuradio/basic_block.h:15,
                 from /usr/include/gnuradio/block.h:15,
                 from /usr/include/gnuradio/sync_block.h:15,
                 from /usr/include/gnuradio/sync_decimator.h:15,
                 from /opt/gr-dsd/lib/../include/dsd/dsd_block_ff.h:11,
                 from /opt/gr-dsd/lib/dsd_block_ff_impl.h:11,
                 from /opt/gr-dsd/lib/dsd_block_ff_impl.cc:12:
/usr/include/features.h:346: note: this is the location of the previous definition
  346 | # define __USE_XOPEN    1
      | 
[ 88%] Linking CXX shared library libgnuradio-dsd.so
/usr/bin/ld: ../dsd/libdsd.a(dsd_main.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(dsd_mbe.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(dsd_serial.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(p25p1_heuristics.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(dsd_audio.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(dsd_file.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(dsd_frame.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(dsd_frame_sync.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(dsd_symbol.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(dsd_upsample.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(dstar.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(nxdn_data.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(nxdn_voice.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(p25p1_hdu.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(p25p1_ldu1.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(p25p1_ldu2.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(p25p1_tdu.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(p25p1_tdulc.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(provoice.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(x2tdma_data.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(x2tdma_voice.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(dmr_data.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(dmr_voice.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(dsd_dibit.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(p25_lcw.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
/usr/bin/ld: ../dsd/libdsd.a(p25p1_ldu.c.o):(.bss+0x0): multiple definition of `exitflag'; CMakeFiles/gnuradio-dsd.dir/dsd_block_ff_impl.cc.o:(.bss+0x0): first defined here
collect2: error: ld returned 1 exit status
make[2]: *** [lib/CMakeFiles/gnuradio-dsd.dir/build.make:113: lib/libgnuradio-dsd.so.1.0.0.0] Error 1
make[1]: *** [CMakeFiles/Makefile2:271: lib/CMakeFiles/gnuradio-dsd.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
```

**Comment by argilo** [comment]
It looks like this was fixed upstream in https://github.com/szechyjs/dsd/pull/105. I'll pull that in soon.

**Comment by argilo** [comment]
This should be fixed in f0493ffe7ce9a6faebc4bfbeb58001cf98ef1662. Let me know if you still have any trouble building.
